pkg/core/object/errors.go

```diff
@@ -1,8 +1,12 @@
 package object
 
-import "errors"
+import (
+	"errors"
+
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/util/logicerr"
+)
 
 // ErrObjectIsExpired is returned when the requested object's
 // epoch is less than the current one. Such objects are considered
 // as removed and should not be returned from the Storage Engine.
-var ErrObjectIsExpired = errors.New("object is expired")
+var ErrObjectIsExpired = logicerr.Wrap(errors.New("object is expired"))
```

Comment:
Lets replace this eror to `local_object_storage` space (here or later)? Seems pretty strange to see it here.

Reply:
Fixed